packages/website/pages/faq.js

```diff
@@ -1,13 +1,102 @@
-import { useEffect } from 'react';
+import { useEffect, useState } from 'react';
 
 import FAQPageData from '../content/pages/faq.json';
 import BlockBuilder from '../components/blockbuilder/blockbuilder.js';
 import { initFloaterAnimations } from '../lib/floater-animations.js';
 import GeneralPageData from '../content/pages/general.json';
+import AccordionBlock from '../components/accordionblock/accordionblock';
 
 export default function Home() {
+  const [faqSections, setFaqSections] = useState();
+  /** @type [any, null | any] */
+  const [propsToParseAsMarkdown, setPropsToParseAsMarkdown] = useState([]);
   const sections = FAQPageData.page_content;
   const animations = FAQPageData.floater_animations;
+  const netlifyCMSHost = process.env.NEXT_PUBLIC_NETLIFY_CMS_ENDPOINT || 'https://blog.web3.storage';
```

Comment:
can we remove the default value of `https://blog.web3.storage` here?  i'm concerned that we may end up forgetting to add it in the env config.  or, perhaps that doesn't matter since this is the correct URL in prod?

Reply:
Go that fallback removed.

Reply:
I added back the fallback.  It's consistent with how it's handled in nft.storage, and I'm not sure we need to manage an env var for this since we're using the same domain across environments.  We also link directly to blog.web3.storage in other parts of the site, so it's unlikely we'll ever want to change this.